modules/doc/content/applications/thermal_hydraulics.md

```diff
@@ -0,0 +1,132 @@
+| [Scalar Kernels System](syntax/ScalarKernels/index.md)                  | 0D ODE                                 | Incompressible                                                                                                                                               | 0D                                                                 | 10                    | 1 second        | Bernoulli-like formulations for pressure drop                                                                                                                                                          |
+
+## Open-Source Applications for Advanced Engineering Modeling
+
+Open-source applications for thermal-fluid modeling provide additional capabilities.
+
+| Application Name                                               | Based On                  | Added Features                                                                                                                                         |
+| :------------------------------------------------------------- | :------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Cardinal](https://cardinal.cels.anl.gov/) | [NekRS](https://github.com/Nek5000/nekRS) CFD      | CPU and GPU capabilities for RANS, LES, and DNS. Additional features include Lagrangian particle transport, an ALE mesh solver, overset meshes, and more.                                       |
+
+
+## NCRC Applications for Advanced Engineering Modeling
+
+Via the [Nuclear Computational Resource Center (NCRC)](https://inl.gov/ncrc/), several additional applications are supported for enhanced engineering modeling. See [help/inl/applications.md] for more information.
+
+| Application Name                                               | Based On                  | Added Features                                                                                                                                         |
+| :------------------------------------------------------------- | :------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Pronghorn](https://pronghorn-dev.hpc.inl.gov/site/index.html) | Navier-Stokes Module      | Export controlled correlations for pressure drop, heat exchange, and mass transfer in advanced nuclear reactors.                                       |
+| SAM                                                            | MOOSE Framework           | Additional lumped parameters components for realistic plant modeling and proprietary correlations for pressure drop, heat exchange, and mass transfer. |
```

Comment:
@joshuahansel Thanks for putting this page together! Can you include this link for SAM for the time being? We'll be working on updating the webpage soon, but at least it has @rui-hu's contact info. @lingzou FYI

```suggestion
| [SAM](https://www.anl.gov/nse/system-analysis-module)                                                            | MOOSE Framework           | Additional physics and component models for realistic plant modeling and reactor safety analysis.  |
```

Reply:
thanks for the link @travismui !

Reply:
Thanks! That's good that SAM has a public-facing page - we need that for some of the other apps.